crates/sui-source-validation-service/src/lib.rs

```diff
@@ -41,7 +56,26 @@ pub async fn verify_package(
         .map_err(anyhow::Error::from)
 }
 
-pub async fn initialize(
+pub fn parse_config(config_path: impl AsRef<Path>) -> anyhow::Result<Config> {
+    let contents = fs::read_to_string(config_path)?;
+    toml::from_str(&contents).map_err(anyhow::Error::from)
```

Comment:
...come to think of it, I think `?` already does type conversion on the error, so the idiomatic way to do this is as follows:

```suggestion
    Ok(toml::from_str(&contents)?)
```

Reply:
Oh I like that! Good catch!